### PR TITLE
fix: Add tenant id in config functions

### PR DESCRIPTION
**Pull request by KShivendu**
## Summary of change

Add tenant id in config functions

## Related issues

-   https://github.com/supertokens/supertokens-node/pull/627/
-  https://github.com/supertokens/supertokens-node/pull/629 (Included in the same PR because it happened naturally when I was fixing linter failures. Just 3 lines of change)

## Checklist for important updates

-   [ ] Changelog has been updated
-   [ ] `coreDriverInterfaceSupported.json` file has been updated (if needed)
    -   Along with the associated array in `supertokens_python/constants.py`
-   [ ] `frontendDriverInterfaceSupported.json` file has been updated (if needed)
-   [ ] Changes to the version if needed
    -   In `setup.py`
    -   In `supertokens_python/constants.py`
-   [ ] Had installed and ran the pre-commit hook
-   [ ] Issue this PR against the latest non released version branch.
    -   To know which one it is, run find the latest released tag (`git tag`) in the format `vX.Y.Z`, and then find the latest branch (`git branch --all`) whose `X.Y` is greater than the latest released tag.
    -   If no such branch exists, then create one from the latest released branch.
-   [ ] If have added a new web framework, update the `supertokens_python/utils.py` file to include that in the `FRAMEWORKS` variable
-   [ ] If added a new recipe that has a User type with extra info, then be sure to change the User type in supertokens_python/types.py
-   [ ] Make sure that `syncio` / `asyncio` functions are consistent.

### CODE REVIEW

File: supertokens_python/recipe/emailverification/recipe.py [hunk 1]
```diff
@@ -239,7 +230,7 @@ def callback():
                 PostSTInitCallbacks.add_post_init_callback(callback)
 
                 return EmailVerificationRecipe.__instance
-            raise_general_exception(
+            return raise_general_exception(
```
Comment (rishabhpoddar) [inline supertokens_python/recipe/emailverification/recipe.py:233]: why do you need this return given that the type of raise_general_exception has NoReturn?

File: supertokens_python/recipe/thirdpartyemailpassword/asyncio/__init__.py [hunk 2]
```diff
@@ -62,6 +63,29 @@ async def get_user_by_third_party_info(
     )
 
 
+async def thirdparty_sign_in_up(
```
Comment (rishabhpoddar) [inline supertokens_python/recipe/thirdpartyemailpassword/asyncio/__init__.py:66]: this function is not to be exposed in all the third party related recipes. It's there in the node PR, but it is removed later on - im not sure if it's removed in a PR that is listed.. so remove it here

File: supertokens_python/recipe/thirdpartyemailpassword/syncio/__init__.py [hunk 3]
```diff
@@ -48,6 +49,32 @@ def get_user_by_third_party_info(
     )
 
 
+def thirdparty_sign_in_up(
```
Comment (rishabhpoddar) [inline supertokens_python/recipe/thirdpartyemailpassword/syncio/__init__.py:52]: this is not needed

**Review by rishabhpoddar — CHANGES_REQUESTED** [review]

File: supertokens_python/recipe/thirdpartyemailpassword/syncio/__init__.py (hunk 3 above)
Comment (KShivendu) [reply supertokens_python/recipe/thirdpartyemailpassword/syncio/__init__.py]: Done

File: supertokens_python/recipe/thirdpartyemailpassword/asyncio/__init__.py (hunk 2 above)
Comment (KShivendu) [reply supertokens_python/recipe/thirdpartyemailpassword/asyncio/__init__.py]: Done

File: supertokens_python/recipe/emailverification/recipe.py (hunk 1 above)
Comment (KShivendu) [reply supertokens_python/recipe/emailverification/recipe.py]: Done

**Pull request merged**